src/main/java/io/appform/functionmetrics/FunctionMetricsManager.java

```diff
@@ -30,10 +30,11 @@ public static void initialize(String packageName, MetricRegistry registry) {
             return Optional.empty();
         }
         return Optional.of(registry.timer(
-                String.format("%s.%s.%s.%s",
+                String.format("%s.%s.%s.%s.%s",
```

Comment:
This cannot be done here.
- This will change existing metrics and add something like ".null" if param is not present.
- What will happen if the value has spaces and or some random unicode chars/punctuation, dots etc in the middle?